doc/maintainers.txt

```diff
@@ -72,6 +72,7 @@ Benoît BROSSE: Benoit dot BROSSE at ingenico dot com
 TranslatorGerman
 Peter Grotrian: Peter dot Grotrian at pdv-FS dot de
 Jens Seidel: jensseidel at users dot sf dot net
+Jens Kallup: kallup at web dot de
```

Comment:
This is meant for language translators. You added your name, but I see no changes to `translator_de.h`. Ideally make a separate pull request where you add the missing translation to `translator_de.h` and then you can also add your name here if you plan to help out with future translations as well.

Reply:
@doxygen related #10839